src/LiteCore/test/LiteCore.Tests.Shared/ScopeCollectionTest.cs

```diff
@@ -0,0 +1,574 @@
+                    for (uint i = 1; i <= Native.FLArray_Count((FLArray*)arrColl); i++) {
+                        var collStr = (string)FLSliceExtensions.ToObject(Native.FLArray_Get((FLArray*)arrColl, i-1));
+                        collStr.Should().Be(collName + i, "Because Scope contains all it's collections' name.");
+                        cnt++;
+                    }
+
+                    cnt.Should().Be(2);
+                    Native.FLValue_Release((FLValue*)arrColl);
+
+                    var arrCol2 = Native.c4db_collectionNames(Db, scopeName + 2);
+                    var collStr2 = (string)FLSliceExtensions.ToObject(Native.FLArray_Get((FLArray*)arrCol2, 0));
+                    collStr2.Should().Be(collName + 3, "Because Scope contains it's collection's name.");
+                    Native.FLValue_Release((FLValue*)arrCol2);
+
+                    var arrScope = Native.c4db_scopeNames(Db);
+                    for (uint i = 1; i <= Native.FLArray_Count((FLArray*)arrScope); i++) {
+                        var scopeStr = (string)FLSliceExtensions.ToObject(Native.FLArray_Get((FLArray*)arrScope, i-1));
+                        if((i-1) == 0) 
+                            scopeStr.Should().Be(Database._defaultScopeName, "Because the first socpe in db is default scope.");
+                        else
+                            scopeStr.Should().Be(scopeName + (i-1), "Because Db contains the scopes just added.");
+                    }
+
+                    Native.FLValue_Release((FLValue*)arrScope);
+
+                    var deleteSuccessful = (bool)LiteCoreBridge.Check(err =>
+                    {
+                        return Native.c4db_deleteCollection(Db, collectionSpecA, err);
+                    });
+                    deleteSuccessful.Should().BeTrue("Deleting collection newColl successful.");
+
+                    cnt = 0;
+                    arrColl = Native.c4db_collectionNames(Db, scopeName + 1);
+                    for (uint i = 1; i <= Native.FLArray_Count((FLArray*)arrColl); i++)
+                    {
+                        var collStr = (string)FLSliceExtensions.ToObject(Native.FLArray_Get((FLArray*)arrColl, i - 1));
+                        collStr.Should().Be(collName + (i + 1), "Because the deleted collection is no longer found in scope.");
+                        cnt++;
+                    }
+
+                    cnt.Should().Be(1, "Because collection collName1 is deleted.");
+                    Native.FLValue_Release((FLValue*)arrColl);
+                };
+            });
+        }
+
+        [Fact]
+        public void TestDefaultCollectionInfo()
+        {
+            RunTestVariants(() => {
+                Native.c4coll_getDocumentCount(DefaultColl).Should().Be(0, "because the default collection is empty");
+                Native.c4coll_getLastSequence(DefaultColl).Should().Be(0, "because the default collection is empty");
+                var publicID = new C4UUID();
```

Comment:
This line and the rest of the test are not relevant to the test itself.  In fact I think this whole test can be merged into the first test (TestDefaultCollection)